runelite-client/src/main/java/net/runelite/client/plugins/specorb/SpecOrbOverlay.java

```diff
@@ -56,21 +56,23 @@
 	private static final int RUN_ORB_X_RESIZABLE = 12;
 	private static final int RUN_ORB_Y_RESIZABLE = 103;
 
-	private static final Color SPECIAL_ORB_BACKGROUND_COLOR = new Color(51, 102, 255);
-	private static final Color SPECIAL_ORB_RECHARGE_COLOR = new Color(153, 204, 255, 50);
+	private static Color SPECIAL_ORB_BACKGROUND_COLOR;
```

Comment:
These two fields should be non-static and also probably not uppercase as they are no longer final.